packages/payment-integration-api/src/common/utility/amount-transformer.ts

```diff
@@ -0,0 +1,7 @@
+export default class AmountTransformer {
```

Comment:
Does this need to be in shared package, or this can be moved inside affirm package?

Reply:
As I see, we use this in several places, not only for affirm, so will be better to move it to a shared package.
But I think will be better to do this in another PR.

Reply:
I will create separate ticket and PR and share the links here

Reply:
I created ticket https://bigcommercecloud.atlassian.net/browse/PI-1604 for moving AmountTransfer

Reply:
PR https://github.com/bigcommerce/checkout-sdk-js/pull/2358